index.html

```diff
@@ -615,6 +577,32 @@ <h2>
             <li>Resolve <var>p</var>.
             </li>
           </ol>
+          <p class="note" title="Payment Handler Install">
+            Calling <a>set()</a> for the first time is equivalent to installing
+            a payment handler for this site.
+          </p>
```

Comment:
@rsolomakhin, I'm not sure what that statement means or its impact. Could you explain a bit more?

Reply:
Sure, @ianbjacobs. Chrome will persist a list `L` of origins that are payment handlers. When an origin calls `instruments.set()`, Chrome will check whether this origin is in the list `L`.

If the origin is in the list `L`, then proceed with the rest of the algorithm, because this is an update of instruments in an existing payment handler.

If the origin is _not_ in the list `L`, then use heuristics to determine whether to deny the `.set()` promise outright (for bad actors, for example), allow the `.set()` algorithm to proceed silently (due to user settings, for example), or prompt the user via a dialog with a question like "Install payment app from origin X?" If the `.set()` algorithm runs to successful completion, then add the origin to `L`.

Reply:
Hi @rsolomakhin,

If I understand, the way that one registers a handler is now through set(). If that's the case, then I suggest as an alternative to your note, the following text at the very beginning of section 3. (Registration):

 <p>One registers a payment handler with the user agent when assigning the first <a>PaymentInstrument</a> to it through the <a>set()</a> method.</p>